extension/content_script.js

```diff
@@ -11,8 +11,9 @@ if (window[namespace] === true) {
 
 // Keeps track of the pointed text. Initialized by end of script load.
 var tracker = null;
-// Creates display for time remaining and reading speed at top of page. Initialized by end of script load.
```

Comment:
comment got deleted?

Reply:
Hmm *look of confusion, adds back in comment*